### ISSUE TITLE
Missing Js lib

### ISSUE THREAD

**Issue by cuneyitkiris** [ejwa/gitinspector#120]
All the reports that generated by gitinspector contain below errors:
`1: Uncaught ReferenceError: None is not defined `
`2: Uncaught TypeError: $(...).tablesorter is not a function`
1. error location 

```html
</script>
		<script type="application/javascript">None</script>
		<script type="application/javascript">/* Plugin for jQuery for working with colors.
 * 
 * Version 1.1.
..
```

2.error location:
```javascript
...
$("table#changes, table#blame").tablesorter({
        sortList: [[0, 0]],
...
```
Probably 2. error depends to 1. one. Apparently, user sort in the list doesn't work and user highlight when mouse moving over pie-graph . 
Report params :
```bash
gitinspector -F htmlembedded --grading -l -m -r -x Installer -x libs --since=12/07/16 > ${file}.html
```
My system specs:
```bash
OS: Linux  4.4.0-28-generic #47-Ubuntu SMP Fri Jun 24 10:09:13 UTC 2016 x86 64
```
and gitinspector info
```bash
$ dpkg -s 
Package: gitinspector
Status: install ok installed
Priority: optional
Section: vcs
Installed-Size: 405
....
Architecture: all
Version: 0.4.3+dfsg-1
....
Depends: python:any (<< 2.8), python:any (>= 2.7.5-5~), git, libjs-jquery, libjs-jquery-flot, libjs-jquery-tablesorter
```
and dependency versions:
```bash
$ dpkg -s libjs-jquery-tablesorter |grep Version
Version: 10-2ubuntu2
$ dpkg -s libjs-jquery |grep Version
Version: 1.11.3+dfsg-4
$ dpkg -s libjs-jquery-flot |grep Version
Version: 0.8.3+dfsg-1
```

**Comment by adam-waldenberg** [comment]
Hi. The official Debian package is maintained by @ChristianKastner. Debian-specific changes, which I believe is what's causing this, are hosted in their repositories. As this is a debian-related problem, could you please report it to the Debian bug tracker? You can find more information about it [here](https://www.debian.org/Bugs/Reporting).

In the meantime, you can use the debian packages available under [releases](https://github.com/ejwa/gitinspector/releases). These package the tablesorter plugin directly.

**Comment by ChristianKastner** [comment]
Hi,

I can confirm that this is a Debian-specific bug (and also affecting derivatives such as Ubuntu) caused by a typo in a patch. The patch rewrites paths to use local copies of jQuery libraries; it introduced a typo in a variable name and that broke the including of tablesorter.js.

I reported this as https://bugs.debian.org/831817 for tracking purposes.

I have just uploaded a fixed package to Debian unstable. I'll ask Ubuntu to sync this new version.

Thank you for the report1
Christian
